sdk/src/main/scala/com/horizen/account/state/AccountState.scala

```diff
@@ -111,14 +116,52 @@ class AccountState(val params: NetworkParams,
       stateView.applyMainchainBlockReferenceData(mcBlockRefData).get
     }
 
-    for (tx <- mod.sidechainTransactions) {
-      stateView.applyTransaction(tx).get
+    // get also list of receipts, useful for computing the receiptRoot hash
+    val receiptList = new ListBuffer[EthereumReceipt]()
+    val blockNumber = stateView.getHeight
```

Comment:
probably `stateView.getHeight + 1`

Reply:
We already updated the height at this point

Reply:
The problem was that at the genesis initialization phase we incremented height twice, I added a check that the block hash used as version is not the null array. The correct line here is `stateView.getHeight + 1` as you pointed out.